src/pmpo_c.cpp

```diff
@@ -346,11 +345,10 @@ void polympo_setMeshNumElms_f(MPMesh_ptr p_mpmesh, const int numElms){
   p_mesh->setElm2ElmConn(elm2Elm);
 }
 
-int polympo_getMeshNumElms_f(MPMesh_ptr p_mpmesh) {
+void polympo_getMeshNumElements(MPMesh_ptr p_mpmesh, int & numElms) {
```

Comment:
The suffix '_f' should be added since the function is connected to a fortran function. 

```suggestion
void polympo_getMeshNumElements_f(MPMesh_ptr p_mpmesh, int & numElms) {
```